pkg/intel/metadata/fit/ent_startup_ac_module_entry.go

```diff
@@ -18,11 +18,11 @@ import (
 )
 
 // EntrySACM represents a FIT entry of type "Startup AC Module Entry" (0x02)
-type EntrySACM struct{ EntryBase }
```

Comment:
SACM as a name b/c it matches the many docs. Many you can have a comment to the effect that 
SACM
means
Startup Anchor Cove Module

or some such.

Reply:
I will just revert this change, it isn't necessary. It helped me to change it when I worked on this because I cannot remember all those damn acronyms.

Reply:
OTOH, we also have `EntryDiagnosticACM` - so ... I think going with `EntryStartupACM` is actually sensible and I'll adjust the docs, too. I didn't know that Go strongly couples doc semantics with identifiers.